docs/releases/release-8-3.md

```diff
@@ -0,0 +1,417 @@
+
+### Additional packages updated or added
+
+[Additional packages](../../management/additional-packages) are packages made available by the XCP-ng team directly in our RPM repositories, for easy installation and update on XCP-ng hosts.
+
+- [Alternate kernel](../../installation/hardware#-alternate-kernel) updated to version 4.19.316 (as of XCP-ng 8.3 release).
+- New [alternate driver packages](../../installation/hardware#-alternate-drivers): `intel-i40e-alt`, `intel-ice-alt`, `tg3-module-alt`, and `mlx4-modules-alt`. Various other alternate driver packages were also updated. See [XCP-ng Drivers](https://github.com/xcp-ng/xcp/wiki/Drivers).
+- `zfs` updated to version 2.1.15. ⚠️ Existing ZFS pools may need to be upgraded using `zpool upgrade`. Check ZFS documentation.
+- `netdata` updated to version 1.44.3.
+- `lsscsi` added to the XCP-ng repositories.
+
+## Status of XOSTOR in XCP-ng 8.3
+
+As of release, XOSTOR (our hyperconverged storage solution based on LINSTOR) is available in XCP-ng 8.3 but is still considered **beta**.
+
+Therefore, upgrading from XCP-ng 8.2.1 with XOSTOR to XCP-ng 8.3 is not yet supported. We will provide upgrade instructions once XOSTOR reaches a stable release on XCP-ng 8.3.
+
+## Deprecations and removals
+
+### PV virtualization mode
+
+The old `PV` (paravirtualized) virtualization mode is now completely removed.
+
+Official support for PV guests stopped with the release of XCP-ng 8.1, on 2020-03-31, but 64-bit PV guests still ran on XCP-ng 8.2.1 despite the deprecation.
+
+In XCP-ng 8.3, they won't run at all.
+
+If you still have PV guests, you need to convert them to HVM, and fix the bootloader because the boot process differs between PV and HVM.
+
+[This blog post](https://xcp-ng.org/blog/2022/01/17/removing-support-for-32-bit-pv-guests/) from early 2022 when we announced the removal of 32-bit PV support may still contain information useful for users needing to make the transition.
+
+### Legacy partitioning layout
+
+XCP-ng 8.3 doesn't support the old partition layout where the system partition only took 4 GB on disk, and GPT partitioning is now mandatory. If you need to upgrade a pool whose hosts still use this partitioning scheme, what to do depends on the exact situation.
+
+* If the partitioning type is GPT, see [using the new partitions](../../installation/upgrade/#using-the-new-partitions).
+* If the partitioning type is MBR, a fresh installation is necessary. The presence of a Dell Utility Partition also prevents upgrading (and is usually coupled with an MBR partitioning type).
+
+### Nested virtualization (though not on purpose)
+
+Upgrading Xen from version 13 to version 17 had a lot of benefits, but it came with one drawback: nested virtualization, which had always been experimental but was at least working in some useful situations (such as testing XCP-ng inside XCP-ng), is now non-functional due to fundamental changes in the codebase.
```

Comment:
why not use 4.13 and 4.17 instead of version 13 and version 17 ?

Reply:
I don't know :sweat:

Reply:
Fixed